include/appfwk/ThreadHelper.hpp

```diff
@@ -88,7 +88,7 @@ class ThreadHelper
    * @brief Start the working thread (which executes the do_work() function)
    * @throws ThreadingIssue if the thread is already running
    */
-  void start_working_thread()
+  void start_working_thread(const std::string name="noname")
```

Comment:
I would make this either a `const std::string&` or just `std::string`

Reply:
I've changed the code according to your suggestion.